eox_core/edxapp_wrapper/backends/users_j_v1.py

```diff
@@ -69,6 +70,35 @@ def check_edxapp_account_conflicts(email, username):
     return conflicts
 
 
+class EdnxAccountCreationForm(AccountCreationForm):
+    """
+    A form to extend the behaviour of the AccountCreationForm.
+    For now the purpose of this form is to allow to make the
+    password optional if the flag 'skip_password' is True.
+
+    This form it's currently only used for validation, not rendering.
+    """
+
+    def __init__(  # pylint:disable=too-many-arguments
+            self,
+            data=None,
+            extra_fields=None,
+            extended_profile_fields=None,
+            do_third_party_auth=True,
+            tos_required=True,
+    ):
+        super().__init__(
+            data=data,
+            extra_fields=extra_fields,
+            extended_profile_fields=extended_profile_fields,
+            do_third_party_auth=do_third_party_auth,
+            tos_required=tos_required,
+        )
+
+        if data.pop("skip_password", False):
+            self.fields['password'] = forms.CharField(required=False)
```

Comment:
so, how the password is gonna be set?

Reply:
this case is for accounts with Third-party authentication. 
I added this since it was requested in the [epic](https://3.basecamp.com/3966315/buckets/8050706/todos/3851505459)